lib/vagrant-service-manager/installer.rb

```diff
@@ -0,0 +1,106 @@
+module VagrantPlugins
+  module ServiceManager
+    BINARY_NAME = {
```

Comment:
Why is this constant defined here? This is uses in the handler right? somewhere else as well?